src/persist/src/postgres.rs

```diff
@@ -278,11 +301,10 @@ impl Consensus for PostgresConsensus {
                     SELECT * FROM consensus WHERE shard = $1 AND sequence_number >= $2
                 )";
 
-        let result = {
-            let client = self.client.lock().await;
-            client.execute(&*q, &[&key, &seqno]).await?
-        };
-        if result == 0 {
+        let mut client = self.client.lock().await;
+        let tx = self.tx(&mut client).await?;
+        let result = { tx.execute(&*q, &[&key, &seqno]).await? };
```

Comment:
why the parens?

Reply:
i think this was a leftover from before, removed